internal/distro/rhel8/azure.go

```diff
@@ -128,6 +128,10 @@ func azureRhuiPackageSet(t *imageType) rpmmd.PackageSet {
 func azureSapPackageSet(t *imageType) rpmmd.PackageSet {
 	packageSet := azureCommonPackageSet(t).Append(SapPackageSet(t))
 
+	// The SAP package set needs to remove the alsa-lib exclude so that libcanberra will install
+	// This is not an ideal solution, but is necessary because of how all of this works
+	packageSet.RemoveExclude([]string{"alsa-lib"})
```

Comment:
Sure, that can be done as well. I did it this way because it was a single difference. Adding it to exclude means adding it to all the other users of the package set, but either way is fine with me.